docs/design_flaws.md

```diff
@@ -770,3 +771,11 @@ Edit `GetForestTreeFrame`:
 +	add a
  	ret
 ```
+
+
+## The 6-bit caught level can only record up to level 63
+
+The 6-bit field that is normally used to record the caught level of Pokémon can only record up to level 63 normally.
```

Comment:
```suggestion
The 6-bit field that is normally used to record the caught level of Pokémon can only record up to level 63.
```

Reply:
Also flesh this out a bit, e.g. note how Lugia and Ho-Oh are level 60, and that the other two bits in the byte are used.